packages/flutter/lib/src/material/color_scheme.dart

```diff
@@ -8,8 +8,8 @@ import 'package:flutter/widgets.dart';
 import 'colors.dart';
 import 'theme_data.dart';
 
-/// A set of twelve colors based on the
-/// [Material spec](https://material.io/design/color/the-color-system.html)
+/// A set of colors based on the
+/// [Material spec](https://m3.material.io/styles/color/the-color-system/key-colors-tones)
```

Comment:
Nit: https://m3.material.io/styles/color/the-color-system/color-roles might be a better link

Reply:
I actually debated this with myself for a while. I will update to to that link as it is closer in detail to `ColorScheme`.